packages/ember-data/lib/serializers/json-serializer.js

```diff
@@ -266,6 +266,19 @@ export default Serializer.extend({
   },
 
   /*
+    @method normalizeQueryRecordResponse
+    @param {DS.Store} store
+    @param {DS.Model} primaryModelClass
+    @param {Object} payload
+    @param {String|Number} id
+    @param {String} requestType
+    @return {Object} JSON-API Document
+  */
+  normalizeQueryRecordResponse: function(store, primaryModelClass, payload, id, requestType) {
```

Comment:
Can you add this to the switch in `normalizeResponse`? And maybe add an integration test to make sure this gets called correctly when doing `store.queryRecord()`?